packages/ui/src/components/molecules/Table/TableSearch.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FC } from 'react';
+import styled, { css } from 'styled-components';
+
+import { CrossMark, SearchIcon } from '../../../assets';
+import { Input, Typography } from '../../atoms';
+import { goldenGradient } from '../../utils';
+
+export interface TableSearchProps {
+  onSearch?: (value: string) => void;
+  $borderGold?: boolean;
+  onPostfixIconClick?: () => void;
+}
+
+interface TableStructureProps {
+  $totalHeight: number;
+}
+
+export const TableStructure = styled.div<TableStructureProps>`
+  position: relative;
+  transition: max-height 0.3s ease-out;
+  box-shadow: ${({ theme }) => theme.palette.shadow.popup};
+  border-top-left-radius: 24px;
+  border-top-right-radius: 24px;
+  border-bottom-right-radius: 24px;
+  border-bottom-left-radius: 24px;
+  background: ${({ theme }) => theme.palette.primary.primary};
+  @media ${({ theme }) => theme.screens.lg} {
+    margin-top: 20px;
+    margin-left: 20px;
+    overflow-y: scroll;
+  }
+  margin-left: 0px 20px;
+  z-index: 10;
+`;
+
+export const MainAppBodyWrapper = styled.div`
+  position: relative;
+`;
+
+export const WalletStructure = styled.div`
+  @media ${({ theme }) => theme.screens.lg} {
+    max-height: 900px;
+    padding-right: 8px;
+    margin-right: 8px;
+  }
+  @media ${({ theme }) => theme.screens.xl} {
+    max-height: 1100px;
+    padding-right: 8px;
+    margin-right: 8px;
+  }
+  border-bottom-right-radius: 24px;
+  background: ${({ theme }) => theme.palette.content.content};
+`;
+export const MainAppBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 16px;
+  position: absolute;
+  overflow: auto;
+`;
+
```

Comment:
Why is main app body and wrapper inside `TableSearch` component?

Reply:
Needs to go in a separate file.